azure-pipelines.yml

```diff
@@ -0,0 +1,48 @@
+trigger:
+  branches:
+    include:
+    - master
```

Comment:
Why only master? We'd like to test any PR branch as well.

Reply:
Resolved in a8dd2ef